dataactvalidator/config/sqlrules/fabs43_detached_award_financial_assistance_2.sql

```diff
@@ -8,6 +8,7 @@ SELECT
 FROM detached_award_financial_assistance
```

Comment:
This is about the comment but I can't make a note there. The comment should reflect the rule text so it needs to be updated to match.